client/src/project/List.vue

```diff
@@ -3,7 +3,8 @@
     <h2>Register for {{ event.name }}</h2>
     <div class="row">
       <div class="col">
-        <p>You have already registered a coolest project. Do you want to edit it or do you have another team you want to register?</p>
+        <p v-if="projects.length > 0">You have already registered a coolest project. Do you want to edit it or do you have another team you want to register?</p>
+        <p v-else>You have not registered a coolest project. Do you want to register one?</p>
```

Comment:
We don't refer to projects as "coolest projects". I'd say

> You have not yet registered a project. Click the button below to get started!

/cc @conoramurphy

Reply:
Just noticed this was already discussed on Trello. Never mind so I guess.

Reply:
Ah you're probably right though. "You have already registered a project." is better

Reply:
This is for when they **haven't** registered. So do you mean "You have not registered a project."?

Reply:
Sorry, there are 2 lines above. It should be changed to just "project" instead of "coolest project" in both line 6 & 7.